dbt/include/snowflake/macros/adapters.sql

```diff
@@ -72,14 +72,16 @@
 )
 {% endmacro %}
 
-{% macro snowflake__create_view_as(relation, sql) -%}
+{% macro snowflake__create_view_as_with_temp_flag(relation, sql, temporary=False) -%}
```

Comment:
nit: to disambiguate the macro arg and the value of "temporary" can we rename to `is_temporary`?